api/v1beta1/vmstaticscrape_types.go

```diff
@@ -33,6 +35,7 @@ type TargetEndpoint struct {
 	Path string `json:"path,omitempty"`
 	// HTTP scheme to use for scraping.
 	// +optional
+	// +kubebuilder:validation:Enum=http;https
```

Comment:
Could you also add it to the other Scheme definitions at
`VMPodScrape`, `VMServiceScrape`, `VMNodeScrape`.

Reply:
Sure, done.
And I will check other fields to see if more checks can be done in advance. Maybe replace some sanity checks in webhook.
https://github.com/VictoriaMetrics/operator/blob/ea0c0cefec8bdae4c42bd5f0148da6e3f1889b5f/api/v1beta1/vmagent_types.go#L175-L180
https://github.com/VictoriaMetrics/operator/blob/ea0c0cefec8bdae4c42bd5f0148da6e3f1889b5f/api/v1beta1/vmagent_webhook.go#L52-L55